bi-connectors/PowerBIConnector/docs/AmazonOpenSearchService.md

```diff
@@ -64,3 +64,16 @@
 2. Check if auth credentials are correct.
 3. Check if server is running.
 
+## Limitations and issues
+
+There are known limitations and issues that are tracked by OpenSearch including the items listed below.
+
+| Issue | Description |

```

Comment:
Issues should link to existing GitHub tickets.

Reply:
Updated in e89be25a0afeb0eb8dbe366e461a19f7144c9b6b